### PR TITLE
Stringer interface for EntryNotified type.

### PR DESCRIPTION
**Pull request by mehmettokgoz**
PR closed by Hazelcast automation as no activity (>3 months). Please reopen with comments, if necessary. Thank you for using Hazelcast and your valuable contributions

### CODE REVIEW

**Review by srknzl — APPROVED** [review]

File: events.go [hunk 1]
```diff
@@ -79,6 +107,11 @@ func (e *EntryNotified) EventName() string {
 	return eventEntryNotified
 }
 
+func (e *EntryNotified) String() string {
```
Comment (utku-caglayan) [inline events.go:110]: Can you add a test for this, please?

**Comment by utku-caglayan** [comment]
Also let's cc @yuce since this changes the public API

File: events.go (hunk 1 above)
Comment (mehmettokgoz) [reply events.go:110]: Where can I write test about that? There is `events_tests.go` file but I couldn't be sure.

Comment (utku-caglayan) [reply events.go:110]: `events_tests.go` makes sense 👍

**Pull request closed without merging**